src/ui/components/ObjectLabel.vue

```diff
@@ -50,11 +50,12 @@
 <script>
 import { inject } from 'vue';
 
+import { PREVIEW_ACTION_KEY } from '@/ui/preview/PreviewAction.js';
```

Comment:
I don't think we can have aliased imports due to the JPL codebase @davetsay @jvigliotta

Reply:
This is only true for files that they import directly from source (🙈). @jvigliotta @davetsay which files are we honor-bound to keep webpack-alias-free?

Reply:
Good looking out, @unlikelyzero . We'll fix this on the OMM side. Core should not worry about using or not using aliased imports. As long as we use the same webpack aliases in OMM it will work (🙈+1).